test/src/com/xilinx/rapidwright/design/TestDesignTools.java

```diff
@@ -1224,4 +1224,64 @@ public void testGetConnectionPIPsBiDir() {
         Assertions.assertNotNull(pips);
         Assertions.assertEquals(9, pips.size());
     }
+
+    @Test
+    public void testGetConnectedCells() {
+        Design design = RapidWrightDCP.loadDCP("picoblaze_ooc_X10Y235.dcp");
+        SiteInst si = design.getSiteInstFromSiteName("SLICE_X15Y238");
+        {
+            SitePinInst spi = si.getSitePinInst("E3");
+            Assertions.assertEquals("[processor/data_path_loop[4].arith_logical_lut/LUT6(BEL: E6LUT), processor/data_path_loop[4].arith_logical_lut/LUT5(BEL: E5LUT)]",
+                    DesignTools.getConnectedCells(spi).toString());
```

Comment:
Should these be sorted before comparison to filter out arbitrary hash population compiler noise?